src/views/Login.vue

```diff
@@ -1,85 +1,92 @@
+            <base-input
               v-model="model.email"
               type="email"
               :name="$t('general.name.email')"
               :label="$t('general.name.email')"
               rules="required|email"
               :placeholder="$t('general.placeholder.email')"
-            />                     
-             <base-input 
+            />
+            <base-input
               v-model="model.password"
               type="password"
               :name="$t('general.name.password')"
               :label="$t('general.name.password')"
               rules="required|min:6"
               :placeholder="$t('password')"
             />
-             <router-link class="font-medium text-indigo-600 hover:text-indigo-500 text-sm" to="/forgot-password">
-                  {{ $t('forgotPassword.text') }}
-              </router-link>
-          <div class="w-full flex justify-center">
-            <base-button 
-             type="submit"
-             :disabled="loading === valid"
-             size="lg"
-             color="primary" >
-                   <i
-                v-if="loading"
-                class="text-lg leading-none el-icon-loading"
-              ></i>
-               {{ $t('register.button.label.signIn') }}
-               </base-button>
-          </div>
-        </form>
-         </ValidationObserver>
+            <router-link
+              class="text-sm font-medium text-indigo-600 hover:text-indigo-500"
+              to="/forgot-password"
+            >
+              {{ $t('forgotPassword.text') }}
+            </router-link>
+            <div class="flex justify-center w-full">
+              <base-button
+                type="submit"
+                :disabled="loading === valid"
+                size="lg"
+                color="primary"
+                :loading="loading"
+              >
+                {{ $t('register.button.label.signIn') }}
+              </base-button>
+            </div>
+          </form>
+        </ValidationObserver>
       </div>
     </div>
   </div>
 </template>
 
 <script>
-
 import authService from '@/api/authService';
-import get from 'lodash/get'
+import get from 'lodash/get';
 
 export default {
-  components: { },
+  components: {},
   name: 'Login',
   data() {
     return {
-    loading: false,
-     model: {
-     email:'',
-     password:''
-     },
-    }
+      loading: false,
+      model: {
+        email: '',
+        password: '',
+      },
+    };
   },
   methods: {
     async handleSubmit() {
-    try {
+      try {
         this.loading = true;
-        let {data} =  await authService.login(this.model)
-        const token = get(data, 'token', '')
-        authService.setToken(token)
+        let { data } = await authService.login(this.model);
+        const token = get(data, 'token.plainTextToken', '');
+        authService.setToken(token);
+        this.$store.commit('setUserState', token);
```

Comment:
Same as above, you can use mapActions and mapGetters to access the methods from vuex